### DISCUSSION TITLE
Inviting members

### DISCUSSION THREAD

**Discussion by evetra** [knowledgefutures/pubpub#2893] [Support]
Hello, I have just create a community, but cannot figure out how I can invite others members. I would have expected to be able to enter someone's email in the "Add Members" screen, and for them to receive an invitation. How do I invite others?

**Comment by jeffpooley** [comment]
Hi @evetra, I think you have to point invitees who do not already have PubPub accounts to {your-community-url}/signup. Once they have an account, they will be searchable in the Add Members field.
